src/v2/Apps/Collect/Routes/Collection/Components/CollectionsHubRails/FeaturedCollectionsRails/index.tsx

```diff
@@ -114,21 +98,25 @@ export const FeaturedCollectionEntity: React.FC<FeaturedCollectionEntityProps> =
             })}
           />
         </Flex>
-        <Text variant="subtitle" mt={1} maxWidth={["246px", "100%"]}>
-          <Truncator maxLineCount={1}>{title}</Truncator>
+
+        <Text
+          variant="subtitle"
+          mt={1}
+          maxWidth={["246px", "100%"]}
+          overflowEllipsis
+        >
+          {title}
         </Text>
+
         {price_guidance && (
           <Text variant="small" color="black60">
             {`From $${formattedPrice}`}
           </Text>
         )}
-        {/* TODO: fix this truncation/overflow issue properly, rather than this overflow:hidden Box */}
-        <Box style={{ overflow: "hidden" }}>
-          <ExtendedText mt={1}>
-            <Media lessThan="md">{getTruncatedDescription(4)}</Media>
-            <Media greaterThan="sm">{getTruncatedDescription(3)}</Media>
-          </ExtendedText>
-        </Box>
+
+        <Text variant="text" mt={1}>
+          <ReadMore maxChars={100} content={description} disabled />
```

Comment:
Does `disabled` mean that tapping the **Read More…** link has no effect?

Reply:
Yeah, it shows the "Read More" text but clicking anywhere goes to the collection regardless rather than expanding the text.